definitions/npm/styled-components_v5.x.x/flow_v0.104.x-/styled-components_v5.x.x.js

```diff
@@ -253,7 +253,7 @@ declare module 'styled-components' {
       ...Interpolation<PropsWithTheme<StyleProps, Theme>>[]
     ) => StyledComponent<StyleProps, Theme, V>,
     [[call]]: <StyleProps, Theme>(
-      (props: PropsWithTheme<StyleProps, Theme>) => Interpolation<any>
+      (props: PropsWithTheme<StyleProps, Theme>) => Interpolation<any> // eslint-disable-line flowtype/no-weak-types
     ) => StyledComponent<StyleProps, Theme, V>,
     +attrs: <A: { ... }, StyleProps = {||}, Theme = {||}>(
```

Comment:
Should `StyleProps` and `Theme` use `: {||}` instead of `= {||}`?

Reply:
No, if it used `: {||}` (adding a type, or "narrowing" the type) then it wouldn't accept any object apart from an empty object. The reason we have `= {||}` (default arg) is so people don't have to supply this themselves, since the default case is good enough most of the time. You can read more about it here: https://flow.org/en/docs/types/generics/#syntax-of-generics-